docs/features/cloudevent/receive-cloudevents-job.md

```diff
@@ -0,0 +1,41 @@
+---
+title: "Securely Receive CloudEvents"
+layout: default
+---
+
+# Securely Receive CloudEvents
+
+The `Arcus.BackgroundJobs` library provides a background job to securely receive [CloudEvent](https://github.com/cloudevents/spec)s.
+This allows API's to have asynchronous jobs that allow external parties to forward events to react upon, without exposing a public endpoint.
```

Comment:
I've rephrased it a bit, what do you think?

```suggestion
This allows workloads to asynchronously process event from other components without exposing a public endpoint.

By using Azure Service Bus Topics, we support applications with multiple instances by using a subscription dedicated to every node.
```

Reply:
Done.